exporter/clickhousetracesexporter/feature_manager.go

```diff
@@ -85,11 +92,14 @@ func enableDurationSortFeature(db clickhouse.Conn, options *Options) error {
 		INDEX idx_httpUrl httpUrl TYPE bloom_filter GRANULARITY 4,
 		INDEX idx_httpHost httpHost TYPE bloom_filter GRANULARITY 4,
 		INDEX idx_httpMethod httpMethod TYPE bloom_filter GRANULARITY 4,
-		INDEX idx_timestamp timestamp TYPE minmax GRANULARITY 1
+		INDEX idx_timestamp timestamp TYPE minmax GRANULARITY 1,
+		INDEX idx_rpcMethod rpcMethod TYPE bloom_filter GRANULARITY 4,
+		INDEX idx_responseStatusCode responseStatusCode TYPE set(0) GRANULARITY 1,
 		) ENGINE MergeTree()
 		PARTITION BY toDate(timestamp)
 		ORDER BY (durationNano, timestamp)
-		SETTINGS index_granularity = 8192`, options.primary.TraceDatabase, options.primary.DurationSortTable, options.primary.Cluster))
+		TTL toDateTime(timestamp) + INTERVAL 604800 SECOND DELETE
+		SETTINGS index_granularity = 8192, ttl_only_drop_parts = 1`, options.primary.TraceDatabase, options.primary.DurationSortTable, options.primary.Cluster))
```

Comment:
Don't have to change anything now but you can create independent PRs for different changes.